flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/planner/delegation/hive/HiveParserCalcitePlanner.java

```diff
@@ -2539,9 +2539,9 @@ private RelNode genUDTFPlan(
 
         SqlOperator convertedOperator = convertedCall.getOperator();
         Preconditions.checkState(
-                convertedOperator instanceof SqlUserDefinedTableFunction,
+                convertedOperator instanceof BridgingSqlFunction,
                 "Expect operator to be "
-                        + SqlUserDefinedTableFunction.class.getSimpleName()
+                        + BridgingSqlFunction.class.getSimpleName()
```

Comment:
What's the reason for changing this?

Reply:
Before this pr, the Hive's udtf will be converted to `HiveTableSqlFunction`, which is instance of SqlUserDefinedTableFunction. The old code logic is in  [here](https://github.com/apache/flink/pull/18958/files#diff-ca5db8275fd03a414acc2af23a9114f47791506fb0542986f55f31865f1dbb87L130).
After this pr, it'll then convert to `BridgingSqlFunction`.
So, we need to.change it.